Extensions/EdFi.Ods.Extensions.Sample/Versions/1.0.0/Standard/4.0.0/Artifacts/MsSql/Structure/Ods/0020-EXTENSION-sample-Tables.sql

```diff
@@ -24,11 +24,11 @@ CREATE TABLE [sample].[Bus] (
     ) WITH (PAD_INDEX = OFF, STATISTICS_NORECOMPUTE = OFF, IGNORE_DUP_KEY = OFF, ALLOW_ROW_LOCKS = ON, ALLOW_PAGE_LOCKS = ON) ON [PRIMARY]
 ) ON [PRIMARY]
 GO
-ALTER TABLE [sample].[Bus] ADD CONSTRAINT [Bus_DF_CreateDate] DEFAULT (getdate()) FOR [CreateDate]
+ALTER TABLE [sample].[Bus] ADD CONSTRAINT [Bus_DF_CreateDate] DEFAULT (getutcdate()) FOR [CreateDate]
```

Comment:
These changes to UTC time stamps are unrelated to this story.
Seems that this was recently changed in Metaed.